guides/advanced_guides/installation.md

```diff
@@ -107,3 +107,9 @@ Here is the list of [all the options and how to use them](/guides/advanced_guide
 Updating MeiliSearch is a straightforward process, simply get and install the latest binary with your preferred method (see installation above).
 
 Note that all MeiliSearch versions are not compatible with each other, and you may need to re-index your documents. If you get the error `Cannot open database, expected MeiliSearch engine version: X.X.X, current engine version Y.Y.Y`, delete your database folder (defaults to `data.ms`), and re-index all your documents.
+
+### Restoring your content between two versions
+
+You can use [dumps](/guides/advanced_guides/snapshots_and_dumps.md#dumps) to export your data to a state where it can be imported after updating MeiliSearch. Since the content is exported in a way that guarantees its importability, it needs to be re-indexed. This process can take long if your database is big. Nonetheless, it **guarantees restoring all setting and documents between two updates**.
+
+If you want a complete guide on how to do the transition, please [look at this github issue](https://github.com/meilisearch/MeiliSearch/discussions/1187#discussioncomment-278125).
```

Comment:
```suggestion
Updating MeiliSearch is a straightforward process: simply get and install the latest binary with your preferred method (see [installation](/guides/advanced_guides/installation.md#download-and-launch) above).

Note that prior to our official release (v1.0), MeiliSearch databases are not compatible across versions.

If you want to transfer your database from an outdated MeiliSearch instance to the most recent version, we recommend following [the below guide](/guides/advanced_guides/installation.md#migrating-a-database-to-a-later-version).

If you get the error `Cannot open database, expected MeiliSearch engine version: X.X.X, current engine version Y.Y.Y`, simply delete your database folder (defaults to `data.ms`), and re-index all your documents.

### Migrating a Database to a Later Version

Using [dumps](/guides/advanced_guides/snapshots_and_dumps.md#dumps), you can export your MeiliSearch data—all indexes, documents, and settings contained in your database—into a transferrable state. Then, you can re-import this data after updating MeiliSearch to the latest version.

Since the content is exported in a way that guarantees mobility, it needs to be re-indexed. If your database is large, this process can take a long time. Nonetheless, this process guarantees **to migrate all settings and documents between two different versions of MeiliSearch**.

If you want a complete guide on how to update MeiliSearch on DigitalOcean, please [look at this GitHub issue](https://github.com/meilisearch/MeiliSearch/discussions/1187#discussioncomment-278125).
```

Reply:
Thanks a lot! Just one thing I think should be changed: 

> Note that prior to our official release (v1.0), MeiliSearch databases are not compatible across versions.

Prior to v1.0, databases are not compatible between minor (`v0.*`) updates. Could you update this and I will commit?

Reply:
`Note that prior to our official release (`v1.0`), all minor updates (`v0.X`) are considered breaking. Therefore, **MeiliSearch databases are not compatible across versions** while we are in beta. The only exception to this rule is **patch updates** (e.g. `v0.18.0` is compatible with `v0.18.1`).`
I tried to make it a bit more clear, is this like what you had in mind?

Reply:
Would it maybe be helpful to link to the semantic versioning guidelines?

Reply:
Yes! I don't think adding the sentence about the patch is useful. More so if you link the semver guidelines

Reply:
Removed the sentence about patches. Linking to the semver guidelines feels a little confusing and unnecessary here, so I think I'll wait for the rewrite. Can start the article by explaining that MeiliSearch uses [semantic versioning](...)